cbmc/proofs/scalar_compress_d5/cbmc-viewer.json

```diff
@@ -0,0 +1,7 @@
+{ "expected-missing-functions":
```

Comment:
What do these cbmc-viewer.json files do? Why are they required? Should we be adding them to all proof subdirectories?